contracts/Tokens/VAI/VAIControllerStorage.sol

```diff
@@ -0,0 +1,57 @@
+pragma solidity 0.8.13;
+
+import { IComptroller } from "../../ComptrollerInterface.sol";
+import { ResilientOracleInterface } from "@venusprotocol/oracle/contracts/interfaces/OracleInterface.sol";
+
+contract VAIControllerStorage {
+    /// @notice Fee percent of accrued interest with decimal 18
+    uint256 public treasuryPercent;
+
+    /// @notice The base rate for stability fee
+    uint256 public baseRateMantissa;
+
+    /// @notice The float rate for stability fee
+    uint256 public floatRateMantissa;
+
+    /// @notice VAI Mint Rate as a percentage
+    uint256 public VAIMintRate;
+
+    /// @notice Accumulator of the total earned interest rate since the opening of the market. For example: 0.6 (60%)
+    uint256 public VAIMintIndex;
```

Comment:
I see you changed `vai` to `VAI` everywhere, is this intentional? Do you think we should add modifications to our style guide to explicitly allow names starting with abbreviations to be capitalized?

Reply:
Yes, there was inconsistency with the usage of `VAI` and `vai` in different places, so I made those adjustments for consistency. However, I have now reverted them back to their original state.
https://github.com/VenusProtocol/isolated-pools/pull/345/commits/b41a1132836bd9fc699a077d752f9b3972dc5e16